src/StatsBase.jl

```diff
@@ -130,7 +130,7 @@ module StatsBase
     Histogram,
     hist,
     # histrange,
-    midpoints,
+    # midpoints,    # TODO export when depending on Julia v0.7
```

Comment:
Better put this under a `VERSION` check, so that Femtocleaner fixes it automatically. I think the best approach is to always export the symbol (what is currently done), and conditionally import if from Base on 0.6. Then you can drop the module prefix in the docs.

Reply:
The new commit doesn't correspond to what I suggested. Have you tried that?